packages/boiler-task-selenium/src/util/read-lines.js

```diff
@@ -0,0 +1,23 @@
+// Libraries
+import fs from 'fs';
+// Packages
+import boilerUtils from 'boiler-utils';
+
+
+const {buildLogger, gulpTaskUtils} = boilerUtils;
+const {logError} = gulpTaskUtils;
+const {log} = buildLogger;
+export default function readLines(fp, offset = 50) {
+  try {
+    log(`-------------------- START reading ${fp}`);
+    const lines = fs.readFileSync(fp).toString().split('\n');
+    log([
+      ...lines.slice(0, offset),
+      '....................',
+      ...lines.slice(-offset)
+    ].join('\n'));
+    log(`-------------------- END reading ${fp}`);
+  } catch (err) {
```

Comment:
as mentioned above this can take a `cb` as last argument if you want to `thunk` it. also if you want to skip calling `toString` on the read contents you can pass `'utf8'` or `{encoding: 'utf8'}` as a second option to `readFile` and you can use the async version and call your `cb` when you're done

```js
export default function readLines(fp, offset = 50, cb) {
  readFile(fp, 'utf8', (err, content) => {
    if (err) return cb(err)

    //....log stuff
    cb(null);
  }
}
```